### PR TITLE
feat: transform Testimonial content container on hover

### PR DESCRIPTION
**Pull request by rogervera**
## Adds a scale transform on hover to testimonial content container (not the background)

Design wants this behavior before launch on Monolith.

---

## Merging your changes

When you are ready to merge to master, use the "squash and merge" button in GitHub, and follow the [commit message guide](/README.md#commit-message-guide) to write your commit message.

### CODE REVIEW

File: packages/brand-components/src/Testimonial/styles.module.scss [hunk 1]
```diff
@@ -76,6 +76,10 @@ $background-pattern-offset: px-rem(16px);
 .contentContainer {
   display: grid;
   position: relative;
+  transition: all 0.3s ease-in-out;
```
Comment (JoshuaKGoldberg) [inline packages/brand-components/src/Testimonial/styles.module.scss]: My understanding of `transition` is that it's vaguely good practice to specifically target only the things you want animated - so here that'd be `transform`. Does that sound reasonable? 🤷‍♂

Comment (rogervera) [reply packages/brand-components/src/Testimonial/styles.module.scss]: Ahh, so `transition: transform 0.3s ease-in-out;` ?

Comment (rogervera) [reply packages/brand-components/src/Testimonial/styles.module.scss]: Makes total sense

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@0.7.1-alpha.ce2bed.1+ce2bed46<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5e7d14dc00022d09508b65f4--codecademy-styleguide.netlify.com>https://5e7d14dc00022d09508b65f4--codecademy-styleguide.netlify.com</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5e7d14dc00022d09508b65f4>Deploy Logs</a></p>

**Pull request merged**